doc/user/layouts/shortcodes/configuration-parameters.html

```diff
@@ -48,6 +48,8 @@
 `max_sources`                               | `25`                      | The maximum number of sources in the region, across all schemas.                                                                                                       | [Contact support]
 `max_tables`                                | `25`                      | The maximum number of tables in the region, across all schemas                                                                                                         | [Contact support]
 `mz_version`                                | Version-dependent         | Shows the Materialize server version.                                                                                                                                  | No
+`real_time_recency`                         | `false` | Boolean flag indicating whether [real-time recency](/get-started/isolation-level/#real-time-recency) is enabled for the current session. | [Contact support]
+`real_time_recency_timeout`                 | `10 seconds` | Sets the maximum allowed duration of `SELECT` statements that actively use [real-time recency](/get-started/isolation-level/#real-time-recency). | Yes
```

Comment:
```suggestion
`real_time_recency_timeout`                 | `10 seconds` | Sets the maximum allowed duration of `SELECT` statements that actively use [real-time recency](/get-started/isolation-level/#real-time-recency). If this value is specified without units, it is taken as milliseconds. | Yes
```

Reply:
@sploiselle, I assume this behaves similarly to `statement_timeout`, but wanted to confirm that's the case.

Reply:
yep!